src/clib/lib/res_util/subst_list.cpp

```diff
@@ -563,9 +563,11 @@ void subst_list_add_from_string(subst_list_type *subst_list,
         if (key_len < 0) // There is a ' or " string that is not closed.
             util_abort("%s: missing string delimiter in argument: %s\n",
                        __func__, arg_string_orig);
+
         if (key_len == strlen(tmp)) // There is no '=".
-            util_abort("%s: missing '=' in argument: %s\n", __func__,
-                       arg_string_orig);
```

Comment:
There is still some util abort statements not converted. Should we maybe instead aim for reimplementing this in python straight away?

Reply:
I'll do pybind first, then we can see. I entertain the thought of removing the underlying c-code